TraitError: The 'input' trait of a QuadricDecimation instance is 'read only'. + 1 more failing test

```
======================================================================
ERROR: Test triangular surface decimation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/utils.py", line 635, in dec
    return function(*args, **kwargs)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/tests/test_surface.py", line 158, in test_decimate_surface
    _, this_tris = decimate_surface(points, tris, n_tri)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/surface.py", line 761, in decimate_surface
    return _decimate_surface(points, triangles, reduction)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/surface.py", line 725, in _decimate_surface
    decimate = tvtk.QuadricDecimation(input=src, target_reduction=reduction)
  File "/usr/lib/python2.7/dist-packages/tvtk/tvtk_classes.zip/tvtk_classes/quadric_decimation.py", line 63, in __init__
    tvtk_base.TVTKBase.__init__(self, vtk.vtkQuadricDecimation, obj, update, **traits)
  File "/usr/lib/python2.7/dist-packages/tvtk/tvtk_base.py", line 307, in __init__
    super(TVTKBase, self).__init__(**traits)
  File "/usr/lib/python2.7/dist-packages/traits/trait_handlers.py", line 104, in _read_only
    name, class_of( object ) )
TraitError: The 'input' trait of a QuadricDecimation instance is 'read only'.
```
and another one which just reemerged
```
======================================================================
FAIL: Test CombineMarkersModel Traits Model
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/utils.py", line 635, in dec
    return function(*args, **kwargs)
  File "/home/yoh/deb/gits/pkg-exppsy/build-area/python-mne-0.10.1+dfsg/mne/gui/tests/test_kit2fiff_gui.py", line 35, in test_kit2fiff_model
    assert_false(model.can_save)
AssertionError: <bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7f6568c0fad0>> is not false
    '<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7f6568c0fad0>> is not false' = self._formatMessage('<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7f6568c0fad0>> is not false', "%s is not false" % safe_repr(expr))
>>  raise self.failureException('<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7f6568c0fad0>> is not false')
    
```

I have 

```
$> acpolicy mayavi2   
mayavi2:
  Installed: 4.4.3-2
  Candidate: 4.4.3-2
  Version table:
 *** 4.4.3-2 0
        900 http://http.debian.net/debian/ stretch/main amd64 Packages
        600 http://http.debian.net/debian/ sid/main amd64 Packages
        100 /var/lib/dpkg/status
```

--- a/mne/surface.py
+++ b/mne/surface.py
@@ -717,6 +717,7 @@ def _decimate_surface(points, triangles, reduction):
         os.environ['ETS_TOOLKIT'] = 'null'
     try:
         from tvtk.api import tvtk
+        from tvtk.common import configure_input
     except ImportError:
         raise ValueError('This function requires the TVTK package to be '
                          'installed')
@@ -724,7 +725,8 @@ def _decimate_surface(points, triangles, reduction):
         raise ValueError('The triangles refer to undefined points. '
                          'Please check your mesh.')
     src = tvtk.PolyData(points=points, polys=triangles)
-    decimate = tvtk.QuadricDecimation(input=src, target_reduction=reduction)
+    decimate = tvtk.QuadricDecimation(target_reduction=reduction)
+    configure_input(decimate, src)
     decimate.update()
     out = decimate.output
     tris = out.polys.to_array()

